ast/parser_ext.go

```diff
@@ -417,18 +417,26 @@ func ParseImports(input string) ([]*Import, error) {
 // ParseModule returns a parsed Module object.
 // For details on Module objects and their fields, see policy.go.
 // Empty input will return nil, nil.
-func ParseModule(filename, input string) (*Module, error) {
-	stmts, comments, err := ParseStatements(filename, input)
+func ParseModule(filename, input string, popts ...ParserOptions) (*Module, error) {
+	var stmts []Statement
+	var comments []*Comment
+	var annotations []Annotations
+	var err error
+	if len(popts) == 1 {
+		stmts, comments, annotations, err = ParseStatements(filename, input, popts[0]) //Currently, ParserOptions only supports a single option
```

Comment:
💭 The comment confuses me. ParseStatements takes any number of options, doesn't it?

Reply:
Yes, it can. Removed the comment.